torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -72,6 +88,29 @@ RegisterOperators reg({
             };
           }
         }),
+    Operator(
```

Comment:
We should only have a single prim::TensorToNum, which always does the conversion checks. Having both a checked and uncheck version is confusing. Is there any reason they cannot be unified?